.github/workflows/test-swift.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
-    container: swift:5.10
+    container: swift:6.0
```

Comment:
Swift 6 is not in development anymore (It's installed in my current OS by default) and also I'm using it to run the code locally. (But is not as stable as it was 5.10, still I hope it can fix some issues of the pipeline)